quickstart.Dockerfile

```
@@ -1,44 +1,44 @@
 FROM docker.elastic.co/elasticsearch/elasticsearch:8.5.3
 
 # Environment variable
-ENV ARGILLA_LOCAL_AUTH_USERS_DB_FILE=/usr/share/elasticsearch/users.yml
-ENV TEAM_PASSWORD=1234
-ENV ARGILLA_PASSWORD=1234
-ENV TEAM_API_KEY=team.apikey
-ENV ARGILLA_API_KEY=argilla.apikey
+ENV ADMIN_USERNAME=admin
+ENV ADMIN_PASSWORD=11223344
+ENV ADMIN_API_KEY=team.apikey
+
+ENV ANNOTATOR_USERNAME=argilla
+ENV ANNOTATOR_PASSWORD=11223344
+
+ENV ARGILLA_WORKSPACE=$ANNOTATOR_USERNAME
 ENV LOAD_DATASETS=full
 ENV UVICORN_PORT=6900
+
 ENV xpack.security.enabled=false
 ENV cluster.routing.allocation.disk.threshold_enabled=false
 ENV discovery.type=single-node
 ENV ES_JAVA_OPTS=-'Xms512m -Xmx512m'
 
 USER root
 
+COPY scripts/start_quickstart_argilla.sh /
+COPY scripts/load_data.py /
+COPY dist/*.whl /packages/
+
+
 # Install packages
 RUN apt update && \
     apt -y install curl python3.9 python3.9-dev python3.9-distutils gcc gnupg apache2-utils sudo openssl systemctl && \
     curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && \
     python3.9 get-pip.py && \
-    pip3 install datasets
-
-COPY scripts/start_quickstart_argilla.sh /
-COPY scripts/load_data.py /
-COPY dist/*.whl /packages/
-
-# Install argilla
-RUN chmod +x /start_quickstart_argilla.sh && \
+    pip3 install datasets \
+    # Install Argilla
+    && chmod +x /start_quickstart_argilla.sh && \
     for wheel in /packages/*.whl; do pip install "$wheel"[server]; done && \
-    rm -rf /packages
-
-# This line add context to this image. This solution should be improved
-RUN echo -e "{  \"deployment\":  \"quickstart\" }" \
-  > /usr/local/lib/python3.9/dist-packages/argilla/server/static/deployment.json
+    rm -rf /packages && \
+    rm -rf /var/lib/apt/lists/* \
+    # This line add context to this image. This solution should be improved
+    && echo -e "{  \"deployment\":  \"quickstart\" }" \
```

If I'm wrong ignore it:

```suggestion
    rm -rf /var/lib/apt/lists/* && \
    # This line add context to this image. This solution should be improved
    && echo -e "{  \"deployment\":  \"quickstart\" }" \
```